src/main/kotlin/no/nav/klage/oppgave/service/OppgaveKopiService.kt

```diff
@@ -27,8 +29,7 @@ class OppgaveKopiService(
             val existingOppgaveKopi = oppgaveKopiRepository.getOne(oppgaveKopi.id)
             if (existingOppgaveKopi.versjon < oppgaveKopi.versjon) {
 
-                //Easiest way to keep our unique constraints in db
-                oppgaveKopiRepository.delete(existingOppgaveKopi)
+                mergeMetadata(oppgaveKopi.metadata, existingOppgaveKopi.metadata)
```

Comment:
Denne er jeg ingen fan av tror jeg.. Jeg forstår intensjonen, men synes det bærer preg av å være litt "hacky"..

Reply:
slette-logikken kan ikke brukes. den feiler på grunn av koblingstabellen, så da må vi finne en lurere måte

Reply:
Jeg tror ikke jeg synes dette er så hacky egentlig, selv om det ville vært fantastisk om JPA hadde håndtert dette automagisk. På en eller annen måte så må riktig id inn i innkommende metadata.

Reply:
Skal man først begynne å flytte data rundt, så tenker jeg at det er bedre å flytte data inn i det objektet som faktisk er "attached" i JPA-sesjonen, for det vil Hibernate diffe når sesjonen/transaksjonen er ferdig, og så lagre endringene.